src/api/api.ts

```diff
@@ -0,0 +1,49 @@
+import { taskEither as TE } from "fp-ts";
+import { identity } from "fp-ts/lib/function";
+import { BusinessesApi, Business } from "../models/domain";
+import { pipe } from "fp-ts/lib/pipeable";
+import { Type, TypeOf } from "io-ts";
+
+function getApiCall<T>(
+  relativeUrl: string,
+  params: object,
+  decoder: Type<T>
+): TE.TaskEither<unknown, TypeOf<typeof decoder>> {
```

Comment:
This one could just be

```suggestion
): TE.TaskEither<unknown, T> {
```

?